bcl/inc/bc_system.h

```diff
@@ -6,11 +6,23 @@
 typedef enum
 {
     BC_SYSTEM_CLOCK_MSI = 0,
+
     BC_SYSTEM_CLOCK_HSI = 1,
+
     BC_SYSTEM_CLOCK_PLL = 2
 
 } bc_system_clock_t;
 
+typedef enum
+{
+    BC_SYSTEM_CORE_VERSION_UNKNOWN = 0,
+
+    BC_SYSTEM_CORE_VERSION_R1 = 1,
+
+    BC_SYSTEM_CORE_VERSION_R2 = 2
+
+} bc_system_core_version_t;
```

Comment:
Please, change it to `bc_system_core_module_revision_t` (and all related symbols)